content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -200,11 +209,42 @@ spec:
 You may also be able to mutate the incoming Pod, at admission time, to unset
 the `.spec.nodeName` field and to use a node selector instead.
 
+## Admin access
```

Comment:
How about:
```suggestion
## Resources with privilege escalation risks {#admin-access}
```

Reply:
I prefer the shorter "Admin access" because "Resources with privilege escalation risks" focuses too much on the risk (IMHO) and isn't quite correct (it's about requests, not the resources).